# MACHINE LEARNING

## TASK 2


#### Hypothesis testing is a fundamental statistical method used to make inferences about populations based on sample data. It involves testing a hypothesis about a population parameter, such as a mean or proportion, using sample data.



####  Q1) Suppose a child psychologist claims that the average time working mothers spend talking to their children is at least 11 minutes per day. You conduct a random sample of 1000 working mothers and find they spend an average of 11.5 minutes per day talking with their children. Assume prior research suggests the population standard deviation is 2.3 minutes.Conduct a test with a level of significance of alpha = 0.05.

#### 1) Formulate Hypotheses:
Null Hypothesis (H0): The average time working mothers spend talking to their children is 11 minutes or less per day. μ ≤ 11
Alternative Hypothesis (Ha): The average time working mothers spend talking to their children is more than 11 minutes per day. μ > 11

#### 2) Select Significance Level: 
α = 0.05

#### 3) Collect Data: 
Sample size (n) = 1000, Sample mean (x̄) = 11.5 minutes, Population standard deviation (σ) = 2.3 minutesx
ˉ
 −μ
​


#### 4) Compute Test Statistic:

We'll use the z-test because we have the population standard deviation.
The test statistic for a z-test is calculated a z=(x̄-μ)/(σ/√n)

#### 5) Determine Critical Value or P-value:
For a one-tailed test with α = 0.05, the critical value can be found using a standard normal distribution table. Since it's a right-tailed test, we're interested in the upper tail of the distribution.
The critical value for α = 0.05 is approximately 1.64

#### 6.Make Decision:

If the calculated z-value is greater than 1.645, we reject the null hypothesis.
If the calculated z-value is less than or equal to 1.645, we fail to reject the null hypothesis.5.σ


​xˉ−μ

In [1]:
import math

In [5]:
# Calculating z value
Z=(11.5-11)/(2.3/math.sqrt(1000))
Z

6.874516652539955

#### Since 6.87 is greater than 1.645, we reject the null hypothesis.

### Conclusion:
We have sufficient evidence to conclude that the average time working mothers spend talking
to their children is more than 11 minutes per day.

####  Q2) A coffee shop claims that their average wait time for customers is less than 5 minutes. To test this claim, a sample of 40 customers is taken, and their wait times are recorded. The sample mean wait time is found to be 4.6 minutes with a standard deviation of 0.8 minutes. Perform a hypothesis test at a significance level of 0.05 and determine whether there is enough evidence to support the coffee shop's claim.

#### 1.Formulate Hypotheses:
Null Hypothesis (H0): The average wait time for customers is 5 minutes or more. μ ≥ 5
Alternative Hypothesis (Ha): The average wait time for customers is less than 5 minutes. μ < 5

#### 2.Select Significance Level:
α = 0.05

#### 3. Data collectiona Sample size (n) = 40, Sample mean (x̄) = 4.6 minutes, Sample standard deviation (s) = 0.8 minues.5.s:es

#### 4.Compute Test Statistic:
Since the population standard deviation is not given, we'll use the t-test.
The test statistic for a t-test is calculated a t=(x̄-μ)/(s/√n).

#### 5.Determine Critical Value or P-value:
For a one-tailed test with α = 0.05 and degrees of freedom (df) = n - 1 = 39, the critical value can be found using a t-distribution table. Since it's a left-tailed test, we're interested in the lower tail of the distribution.
The critical value for α = 0.05 and df = 39 is approximately -1.68

#### Probable outputs:
If the calculated t-value is less than -1.685, we reject the null hypothesis.
If the calculated t-value is greater than or equal to -1.685, we cannot reject the null hypothesis.5.s:es

In [6]:
# Calculating t value
t=(4.6-5)/(0.8/math.sqrt(40))
t

-3.162277660168382

#### Since -3.16 is less than -1.685, we reject the null hypothesis.

### Conclusion:
We have sufficient evidence to conclude that the average wait time
for customers is less than 5 minutes at a significance level of 0.05.